src/Framework/Interaction/Target.h

```diff
@@ -121,6 +129,7 @@ using TObject::Copy;
   int  fHitQrkPDG;            ///< hit quark PDG code
   bool fHitSeaQrk;            ///< hit quark from sea?
   TLorentzVector * fHitNucP4; ///< hit nucleon 4p
+  TLorentzVector * fHitEleP4; ///< hit electron 4p
```

Comment:
As before let's think if we need a new pointer here, considering that the two things are mutually exclusive.